script.js

```diff
@@ -1,33 +1,36 @@
 var data = {
-  "Рыбы": {
-    "Форель": {},
-    "Щука": {}
+  'Рыбы': {
+    'Форель': {},
+    'Щука': {}
   },
 
-  "Деревья": {
-    "Хвойные": {
-      "Лиственница": {},
-      "Ель": {}
+  'Деревья': {
+    'Хвойные': {
+      'Лиственница': {},
+      'Ель': {}
     },
-    "Цветковые": {
-      "Берёза": {},
-      "Тополь": {}
+    'Цветковые': {
+      'Берёза': {},
+      'Тополь': {}
     }
   }
 };
 
-function createHTML (obj) {
-  let temp = ''
-  let mainHTML = ''
+function isEmpty (obj) {return Object.keys(obj).length > 0 ? false : true}
+
+function createHTML (container, obj) {
+  let mainHTML = document.createElement('ul');
+  
   for (let key in obj) {
```

Comment:
will be better to use Object and array method, but you can leave as is if you do not want change it